draft-ietf-rtcweb-jsep.xml

```diff
@@ -1859,6 +1859,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             This option allows the endpoint to significantly reduce the amount of
             audio bandwidth it receives, at the cost of some fidelity, depending on
             the quality of the remote VAD algorithm.</t>
+            <t>If the "VoiceActivityDetection" option is specified, with a value
+            of "false", the browser MUST negotiate the audio to not suppress any
+            audio. For opus this would use an "usedtx=0" parameter, while for
+            G.729 DTX would be not be used, and ("CN") would not be
+            negotiated. </t>
+            <t> The "VoiceActivityDetection" option does not have any impact on
+            the setting of the vad value in the client to mixer audio level
```

Comment:
suggest quoting "vad"

Perhaps use the word signaling here, e.g.

"the setting of the "vad" value in the signaling of the client to mixer audio level header extension..."